build_tools/kokoro/gcp_ubuntu/cmake/linux/riscv64/test.sh

```diff
@@ -18,41 +18,64 @@ export PS4='[$(date -u "+%T %Z")] '
 # dylib-llvm-aot bytecode codegen.
 export RISCV_TOOLCHAIN_ROOT=${RISCV_RV64_LINUX_TOOLCHAIN_ROOT?}
 
-# Run the binaries under QEMU.
-echo "Test simple_embedding binaries"
-pushd "${BUILD_RISCV_DIR?}/iree/samples/simple_embedding" > /dev/null
 
-"${QEMU_RV64_BIN?}" -cpu rv64,x-v=true,x-k=true,vlen=256,elen=64,vext_spec=v1.0 \
--L "${RISCV_TOOLCHAIN_ROOT?}/sysroot" simple_embedding_dylib
+function generate_dylib_vmfb {
+  local target="${1}"; shift
+  local translate_arg=(
+    -iree-mlir-to-vm-bytecode-module -iree-hal-target-backends=dylib-llvm-aot
+    -iree-llvm-embedded-linker-path=${BUILD_HOST_DIR?}/install/bin/lld
+    -iree-llvm-target-triple=riscv64
+    -iree-llvm-target-cpu=generic-rv64
+    -iree-llvm-target-abi=lp64d
+  )
+  if [[ "${target}" == "tosa" ]]; then
+    local input_file="${1}"; shift
+    iree-import-tflite -o "${BUILD_RISCV_DIR?}/tosa.mlir" "${input_file}"
+    translate_arg+=(
+      -iree-input-type=tosa
+      -iree-llvm-target-cpu-features="+m,+a,+f,+d,+c"
+      "${BUILD_RISCV_DIR?}/tosa.mlir"
+    )
+  elif [[ "${target}" == "tosa-rvv" ]]; then
+    local input_file="${1}"; shift
+    iree-import-tflite -o "${BUILD_RISCV_DIR?}/tosa.mlir" "${input_file}"
+    translate_arg+=(
+      -iree-input-type=tosa
+      -iree-llvm-target-cpu-features="+m,+a,+f,+d,+c,+v"
+      -riscv-v-fixed-length-vector-lmul-max=8
+      -riscv-v-vector-bits-min=256
+      -riscv-v-fixed-length-vector-elen-max=64
+      "${BUILD_RISCV_DIR?}/tosa.mlir"
+    )
+  elif [[ "${target}" == "mhlo" ]]; then
+    translate_arg+=(
+      -iree-input-type=mhlo
+      -iree-llvm-target-cpu-features="+m,+a,+f,+d,+c"
+    )
+  fi
+  "${BUILD_HOST_DIR?}/install/bin/iree-translate" "${translate_arg[@]}" "$@"
+}
 
-"${QEMU_RV64_BIN?}" -cpu rv64,x-v=true,x-k=true,vlen=256,elen=64,vext_spec=v1.0 \
--L "${RISCV_TOOLCHAIN_ROOT?}/sysroot" simple_embedding_embedded_sync
+PIP_INSTALL_DIR="${BUILD_RISCV_DIR?}/pip/"
 
-"${QEMU_RV64_BIN?}" -cpu rv64,x-v=true,x-k=true,vlen=256,elen=64,vext_spec=v1.0 \
--L "${RISCV_TOOLCHAIN_ROOT?}/sysroot" simple_embedding_vmvx_sync
+mkdir -p "${PIP_INSTALL_DIR}"
+pip install --prefix=${PIP_INSTALL_DIR} lit iree-tools-tflite
```

Comment:
I can put `lit` into the riscv docker, but I want to pull the latest `iree-tools-tflite` from pypl for the tosa test. Adding `lit` here is a small overhead. 

Please let me know if you prefer to have `lit` in the docker image.

Reply:
We've already checked out LLVM. I don't think you should need to install lit at all.